include/exadg/convection_diffusion/spatial_discretization/operators/combined_operator.cpp

```diff
@@ -119,7 +122,9 @@ void
 CombinedOperator<dim, Number>::update_after_grid_motion()
 {
   if(operator_data.diffusive_problem)
-    diffusive_kernel->calculate_penalty_parameter(*this->matrix_free, operator_data.dof_index);
+    diffusive_kernel->calculate_penalty_parameter(*this->matrix_free,
+                                                  operator_data.dof_index,
+                                                  operator_data.quad_index);
```

Comment:
With the above comment, this would then read `operator_data.quad_index_standard`.